mteb/tasks/BitextMining/multilingual/TatoebaBitextMining.py

```diff
@@ -147,3 +147,4 @@ class TatoebaBitextMining(AbsTaskBitextMining, CrosslingualTask):
         n_samples={"test": 2000},
         avg_character_length={"test": 39.4},
     )
+    fast_loading = True
```

Comment:
I don't know if this is the best place place to toggle it, but I thought that since it's more of an implementation detail it wouldn't be useful to put it as a metadata

Reply:
It is not really metadata. I would move it to:

```
fast_loading = True
metadata = TaskMetadata( ...
)
```

We might not require a toggle if we just convert all the bitext task to the new format.